_search-plugins/collapse-search.md

```diff
@@ -0,0 +1,332 @@
+---
+layout: default
+title: Collapse search results
+nav_order: 3
+---
+
+# Collapse search results
+
+The collapse parameter in OpenSearch enables you to group search results by a particular field value, returning only the top document within each group. This feature is especially beneficial for reducing redundancy and enhancing performance when handling large datasets. By grouping these documents, you can streamline the search results and focus on unique entries, avoiding duplicates.
```

Comment:
I don't think we should say "enhancing performance" as collapse query will be worse than their counterpart non-collapse query with the same filter. It's more of a utility to remove redundancy of similar results from hits

Reply:
I have clarified this point, thank you.